setup.py

```diff
@@ -75,25 +75,27 @@ def get_version(version_file):
         # that you indicate whether you support Python 2, Python 3 or both.
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3 :: Only",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
         "Programming Language :: Python :: Implementation :: CPython",
     ],
-    python_requires=">=3.7",
+    python_requires=">=3.11",
     # What does your project relate to?
-    keywords="skeleton",
+    keywords=["cyhy", "config"],
     packages=find_packages(where="src"),
     package_dir={"": "src"},
-    package_data={"example": ["data/*.txt"]},
+    package_data={"cyhy_config": ["py.typed"]},
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
-    install_requires=["docopt", "schema", "setuptools >= 24.2.0"],
+    install_requires=[
+        "boto3",
+        "pydantic",
+        "rich",
+        "setuptools >= 24.2.0",
```

Comment:
The stubs libraries for these dependencies [should be added to `.pre-commit-config.yaml`](https://github.com/cisagov/cyhy-config/blob/5624d094e448da3e34b838fc49325e8866913ab9/.pre-commit-config.yaml#L158-L160) if they exist.

Note that [`pydantic` is a little funky](https://github.com/pydantic/pydantic/discussions/9446#discussioncomment-9615466) in this respect.

Also, the `types-docopt` entry in `.pre-commit-config.yaml` should be removed since it is no longer needed.

Reply:
See: 
- a0daf8a4db445fa2ef2e36a6840df102dfc24d90
- 11b3bff54a54f8661feb5b0e83f544b577e8748d

Reply:
Please also remove the `types-docopt` entry in `.pre-commit-config.yaml` as that dependency is no longer being used.

Reply:
Ah, nevermind.  I see that you did it now in commit 9183bbf.